### ISSUE TITLE
Doesn't work with local dependencies

### ISSUE THREAD

**Issue by alexandr-kozlov-clearscale** [zbigg/copy-production-deps#141]
When I have a dependency from local path, i.e.
```
  "dependencies": {
    "api": "file:../api",
    ...
```
`npx copy-production-deps` fails with following error:
```
$ npx copy-production-deps . build/dist
npx: installed 31 in 3.648s
copy-production-deps: failed: Error: Cannot overwrite directory '[PATH]\tools\build\dist\node_modules\api/' with non-directory '[PATH]\tools\node_modules\api/'. Error: Cannot overwrite directory '[PATH]\tools\build\dist\node_modules\api/' with non-directory '[PATH]\tools\node_modules\api/'.
    at C:\Users\Kaa\AppData\Roaming\npm-cache\_npx\6528\node_modules\copy-production-deps\node_modules\fs-extra\lib\util\stat.js:55:19
    at cb (util.js:289:31)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
```

**Comment by ManuelTS** [comment]
This may fail as local dependencies are installed in the `node_modules` dir as symlinks.

**Comment by zbigg** [comment]
Thanks for report, and sorry for late reply.
I confirm this was not even implemented as i wasn't used this feature ...
First attempt to fix in #178

**Comment by zbigg** [comment]
Fixed in [0.2.1](https://github.com/zbigg/copy-production-deps/releases/tag/v0.2.1)
